modules/manage/pages/manage-nodes/modify-services-and-rebalance.adoc

```diff
@@ -0,0 +1,160 @@
+
+You do not have to add or remove a node to add or remove a non-Data Service on a node in a cluster.
+
+NOTE: You cannot add or remove kv (Data Service) using this method.
+
+Adding or removing of the Data Service on an existing node is supported only through adding or removing nodes.
+
+To add a Data Service on a node from the UI:
+
+* xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a New Node and Rebalance] along with enabling the Data Service.
+
+* xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with enabling the Data Service.
+
+To remove a Data Service from an existing node using the UI:
+
+* xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster by disabling the Data Service.
+
+* xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a New Node and Rebalance] along with disabling the Data Service.
+
+== Prerequisites
+
+Before modifying non-data services on nodes, ensure you have the following:
+
+* Full Administrator access.
+* A clear understanding of the current service distribution across the nodes.
+* Service Memory Quota:
+When you add a service that wasn’t earlier present in the cluster, the new service uses the service-specific default memory quota.
+This applies to the Index, Search, Eventing, and Analytics services.
++
+Verify that the memory quota for the new service meets your requirements using one of the following:
+
+** The REST API from xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information].
+** The UI from xref:manage:manage-settings/general-settings.adoc#configure-general-settings-with-the-ui[Configure General Settings with the UI].
++
+NOTE: The default service memory quota is set per server node and is reflected in the `GET /pools/default` REST API response.
+ 
+[#modify-mds-services-from-ui]
+== Modify Services from the UI
+
+To manage non-data services from the UI, follow these steps:
+
+. Open Couchbase Web Console.
+
+. Click [.ui]*Servers*.
+
+. Click btn:[Modify Services].
++
+The [.ui]*Modify Cluster Services* dialog appears, which lists all the nodes in all the clusters and their current services.
++
+--
+Services that can be modified are:
+
+** *index* (Index Service)
+** *n1ql* (Query Service)
+** *fts* (Search Service)
+** *cbas* (Analytics Service)
+** *eventing* (Eventing Service)
+** *backup* (Backup Service).
+--
++
+kv (Data Service) cannot be modified.
+
+. Select the services you want to add for each node and deselect the services you want to remove from each node.
++
+NOTE: Adding or removing any service triggers a rebalance operation.
+
+. Click btn:[Rebalance and Change Services].
++
+If you're only removing services, then rebalancing is triggered immediately.
```

Comment:
If you're only removing services, adding new instances of services that already exist on the cluster, or adding the backup service, then rebalancing is triggered immediately.